docs/pages/example/data-join.html

```diff
@@ -72,30 +72,35 @@
     // https://www.census.gov/geo/maps-data/data/cbf/cbf_state.html
     map.addSource("states", {
         type: "vector",
-        url: "mapbox://mapbox.us_census_states_2015"
+        url: "mapbox://mapbox.10asuvo4",
```

Comment:
I'd rename the tileset to have a self-explanatory name and/or add a comment explaining what data the tileset contains.

Reply:
We can re-upload the tileset with a specific `tilesetid` for improved self-documenting names.